.editorconfig

```diff
@@ -724,6 +728,9 @@ dotnet_diagnostic.CA1417.severity = warning
 # Use 'nameof' in place of string.
 dotnet_diagnostic.CA1507.severity = warning
 
+# Avoid redundant length argument.
+dotnet_diagnostic.CA1514.severity = warning # TODO: Change to warning once using .NET 8 or later.
```

Comment:
The comments imply that these should be something other than warning?

Reply:
Indeed I forgot to actually downgrade them. Duh. Changed to `suggestion` throughout.